modules/exploits/example_webapp.rb

```diff
@@ -26,47 +30,50 @@ def initialize(info = {})
         # vuln type, class. Preferably apply
         # some search optimization so people can actually find the module.
         # We encourage consistency between module name and file name.
-        'Name'           => 'Sample Webapp Exploit',
-        'Description'    => %q(
-            This exploit module illustrates how a vulnerability could be exploited
+        'Name' => 'Sample Webapp Exploit',
+        'Description' => %q{
+          This exploit module illustrates how a vulnerability could be exploited
           in a webapp.
-        ),
-        'License'        => MSF_LICENSE,
+        },
+        'License' => MSF_LICENSE,
         # The place to add your name/handle and email.  Twitter and other contact info isn't handled here.
         # Add reference to additional authors, like those creating original proof of concepts or
         # reference materials.
         # It is also common to comment in who did what (PoC vs metasploit module, etc)
-        'Author'         =>
-          [
-            'h00die <mike@stcyrsecurity.com>', # msf module
-            'researcher' # original PoC, analysis
-          ],
-        'References'     =>
-          [
-            [ 'OSVDB', '12345' ],
-            [ 'EDB', '12345' ],
-            [ 'URL', 'http://www.example.com'],
-            [ 'CVE', '1978-1234']
-          ],
+        'Author' => [
+          'h00die <mike@stcyrsecurity.com>', # msf module
+          'researcher' # original PoC, analysis
+        ],
+        'References' => [
+          [ 'OSVDB', '12345' ],
+          [ 'EDB', '12345' ],
+          [ 'URL', 'http://www.example.com'],
+          [ 'CVE', '1978-1234']
+        ],
         # platform refers to the type of platform.  For webapps, this is typically the language of the webapp.
         # js, php, python, nodejs are common, this will effect what payloads can be matched for the exploit.
         # A full list is available in lib/msf/core/payload/uuid.rb
-        'Platform'       => ['python'],
+        'Platform' => ['python'],
         # from lib/msf/core/module/privileged, denotes if this requires or gives privileged access
-        'Privileged'     => false,
+        'Privileged' => false,
         # from underlying architecture of the system.  typically ARCH_X64 or ARCH_X86, but for webapps typically
         # this is the application language. ARCH_PYTHON, ARCH_PHP, ARCH_JAVA are some examples
         # A full list is available in lib/msf/core/payload/uuid.rb
-        'Arch'           => ARCH_PYTHON,
-        'Targets'        =>
-          [
-            [ 'Automatic Target', {}]
-          ],
-        'DisclosureDate' => '2013-04-01',
+        'Arch' => ARCH_PYTHON,
+        'Targets' => [
+          [ 'Automatic Target', {}]
+        ],
+        'DisclosureDate' => '2020-12-30',
```

Comment:
Why change this? I'm not against it I'm just curious.

Reply:
the year doesn't matter so much, but wanted the months and days to be plainly obvious.  Was the previous january 4th or april 1st? Also, not sure we want it to fall on april fools day :)

need to update `modules/exploits/example.rb` as well for this same thing, missed it there.